project4/index.html

```diff
@@ -0,0 +1,100 @@
+            <li class="nav-item"><a href="#">about us</a></li>
+            <li class="nav-item"><a href="#">buy a ticket</a></li>
+            <li class="nav-item"><a href="#">contact us</a></li>
+            <ul class="hashtag">
+                <li class="hashtag"><a herf="#"><img src="iglogo.png" alt="instagram logo"></a></li>
+
+                <li class="hashtag twitter"><a herf="#"> <img src="twitlogo.png" alt="twitter logo"></a></li>
+                <!--©CROP THE PNG-->
+                <li class="hashtag"><a herf="#"><img src="fb.png" alt="facebook logo"></a></li>
+
+            </ul>
+
+        </ul>
+
+
+    </nav>
+</div>
+<section class="hero">
+    <p>WELCOME,
+        To
+        The Un-Talked</p>
+    <img class="logo" src="Finallogo.svg" alt="">
+    <!--Confrence name already on the page.-->
+    <p>Baltimore MD,
+        May 20th-23rd</p>
+</section>
+
+<section class="opening">
+    <h1>Talks About the Un-Talked </h1>
+    <a>Purchase</a>
+    <p>what are we about?</p>
+    <p>We are a place to congregate and voice the issues we as People of Color face in our own design community.</p>
+    <p>We are here to listen, discuss, and plan on how we can make our work lives safer and so that we are on equal
+        footing to our white counter-parts.</p>
+    <p>Lend us your voice so we can start working towards a better future where all opinions are equally valued.</p>
+</section>
+<section class="speakers">
+    <h2>Speakers</h2>
+    <p>Amber Park</p>
+    <img src="amber.png" alt="Amber Park">
+    <p> After finishing her bachelor’s form MICA in Baltimore, she has gone on to begin her own product design firm in
+        Philadelphia that focuses mainly on package design for brands.</p>
+    <p>Shaika Bugrais</p>
+    <img src="Shay.png" alt="Shaika Bugrais">
+    <p>After finishing her bachelor’s form MICA in Baltimore, she went back to her home country of Kuwait and began her
+        career as a graphic designer mainly focusing on the pride of her Islamic faith.</p>
+    <p>Hana Nagel</p>
+    <p>Hana Nagel is a designer and researcher, with experience ranging from non-profits to start-ups to enterprise. Her
+        work is connected by a commitment to using a research-driven approach to solve problems for people. She is
+        currently a service designer with Element AI, where she helps teams to make the right thing, and to make the
+        thing right.
+        Born in California and raised in Vancouver, Hana firmly believes the West Coast is the best coast. She is
+        currently loving life in Montreal, which she shares with her fiancée, a large cat, a small dog, and 62 plants
+        (and counting).
+    </p>
+    <!--Bio came from https://conveyux.com/speakers/hana-nagel/-->
+    <img src="hana.png" alt="Hana Nagel">
+    <!--Got profile picture for hana from https://www.youtube.com/watch?v=JiBMIR1wZ6k video -->
+</section>
+<section class="schedule">
+    <p>Schedule</p>
```

Comment:
Could also be an `h2` to match the speakers section?
```suggestion
    <h2>Schedule</h2>
```